lib/chutney/TorNet.py

```diff
@@ -763,19 +766,28 @@ def _genRouterKey(self):
                   .format(" ".join(cmdline), stdouterr))
             sys.exit(1)
         self._env['fingerprint'] = fingerprint
-
+       
 
     def _setEd25519Id(self):
-        """Make chutney check for relay microdescriptors before verifying"""
+        """Read the ed25519 identity key for this router, and set up the 'ed25519-id' entry in the Environ"""
         datadir = self._env['dir']
-        key_directory =  os.path.join(datadir, 'keys', "ed25519_master_id_public_key")
-        with open(key_directory ,'rb') as f:
-            f.seek(32)
-            rest_file = f.read()
-            EncodedValue = base64.b64encode(rest_file)
-            ed25519_id = str(EncodedValue)[2:-2]
-            self._env['ed25519-id'] = ed25519_id
-      
+        key_directory = os.path.join(datadir, 'keys', "ed25519_master_id_public_key")
+        if os.path.exists(key_directory):
+            raise ValueError("File does not exit")
```

Comment:
Spelling mistake, which could confuse chutney users:
```suggestion
            raise ValueError("File does not exist")
```

Reply:
It also helps to include the actual size and the expected size in the exception text.